src/main/resources/freemarker/reminder.html

```diff
@@ -21,7 +21,7 @@
         </tr>
         <tr>
             <td style="padding: 15px 15px 0px; font-family: 'Montserrat', sans-serif; font-size: 16px; color: #1F3B50; text-align: justify; line-height: 25px;">
-                Please vote on ${electionType} Review case id ${entityName}. Click on the following link to log your vote:
+                Please vote on Data Access Request: ${entityName}. Click on the following link to log your vote:
```

Comment:
If the `electionType` is no longer being used, should it be deprecated in the template and other locations?

Reply:
Great point, yes, I'll make that change too.